### ISSUE TITLE
Consider adding a named type for the parameter in ngOnChanges

### ISSUE THREAD

**Issue by Tragetaschen** [angular/angular#6292]
Currently, whenever I implement the `OnChanges` interface (and cannot ignore the `changes` prarameter) I have to spell out a lot of characters and create a local, anonymous type:
```ts
function ngOnChanges(changes: {[key: string]: SimpleChange})
```
It would be great if this type was named and exported from angular2

```ts
export interface Changes {
    [key: string]: SimpleChange;
}

function ngOnChanges(changes: Changes)
```

**Comment by laco0416** [comment]
:+1: 

And I want `SimpleChange<T>`.

**Comment by Tragetaschen** [comment]
This was implemented by #8557
